ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/SandboxOnXRunner.scala

```diff
@@ -230,7 +232,7 @@ object SandboxOnXRunner {
   private def buildServicesExecutionContext(
       metrics: Metrics,
       servicesThreadPoolSize: Int,
-  ): ResourceOwner[ExecutionContextExecutorService] =
+  ): ResourceOwner[QueueAwareExecutionContextExecutorService] =
     ResourceOwner
```

Comment:
This binds the metric name to the executor service. This name is then re-associated on line 297.  I don't like that, can we either:
- Add a metric name to QueueAwareExecutionContextExecutorService
- Return a tuple (QueueAwareExecutionContextExecutorService, MetricName)

Reply:
Good point, will go with the first approach as it's simpler and cleaner.